scss/_base_typography.scss

```diff
@@ -153,4 +153,8 @@
     cursor: help;
     text-decoration: none;
   }
+
+  .p-muted-text {
```

Comment:
Can we rename it to `p-text--muted` (to make it consistent with `p-text--small` etc).
I know it makes it incosisntent with `p-muted-heading`, but I guess muted heading could be `p-heading--muted` as well (not in this PR).

Reply:
Oh, I just noticed that it's redefined in separate file. So it should be removed from here.